projects/packages/my-jetpack/_inc/components/product-cards-section/style.module.scss

```diff
@@ -47,13 +47,24 @@
 	}
 }
 
+.cardListTitle,
+.fullStatsCard {
+	margin-bottom: 1.5rem;
+}
+
 .cardlist {
 	> li {
 		margin-bottom: 0; // Reset base style
 	}
 }
 
-.cardListTitle,
-.fullStatsCard {
-	margin-bottom: 1.5rem;
+// Since we're adding so much info into the My Jetpack product cards, here we'll increase
+// the min-width of the cards to 300px, and otherwise wrap around.
+@media screen and (min-width: 599px) and (max-width: 1290px) {
+	ul.cardlist {
+		grid-template-columns: repeat( auto-fill, minmax( 300px, 1fr ) );
+		> li {
+			grid-column-end: auto;
+		}
+	}	
```

Comment:
Thank you for this change! I think everything looks good when scaling except the backup card hides one of the items briefly on a specific screensize and that is no longer necessary. I'll add a maintenance item to fix that

![image](https://github.com/user-attachments/assets/b8b4618e-3fa3-4656-9e2e-2afad5747616)

Reply:
Ahh, ok cool. Thanks! 👍